package.json

```diff
@@ -33,24 +33,19 @@
     "client library"
   ],
   "dependencies": {
-    "async": "^2.0.0",
-    "gtoken": "^1.1.0",
-    "lodash.noop": "^3.0.0",
-    "jws": "^3.1.0",
-    "request": "^2.74.0",
-    "string-template": "^1.0.0"
+    "gtoken": "^1.2.1",
+    "lodash.noop": "^3.0.1",
+    "jws": "^3.1.4",
+    "request": "^2.79.0"
```

Comment:
I went ahead and reverted to `"request": "^2.74.0"` so that decisions about `request v2.79.0` and Node.js 0.10/0.12 don't have to block the 0.10.0 release.